src/main/java/edu/hm/hafner/analysis/LookaheadParser.java

```diff
@@ -151,7 +151,13 @@ private String extractDirectory(final String line, final Pattern makePath) throw
         }
         Matcher makeLineMatcher = makePath.matcher(line);
         if (makeLineMatcher.matches()) {
-            return makeLineMatcher.group("dir");
+            StringBuilder dir = new StringBuilder(makeLineMatcher.group("dir"));
+            if ((dir.charAt(0) == '\'' || dir.charAt(0) == '`') &&
+                    (dir.charAt(dir.length() - 1) == '\'' || dir.charAt(dir.length() - 1) == '`')) {
+                dir.deleteCharAt(0);
+                dir.deleteCharAt(dir.length() - 1);
+            }
```

Comment:
Move that part to a new method `removeHyphen`.

There you can use `StringUtils.stripEnd` and `StringUtils.stripStart`.